ydb/library/yaml_config/yaml_config_parser.cpp

```diff
@@ -210,8 +210,14 @@ namespace NKikimr::NYaml {
             ctx.DisableBuiltinSecurity = GetBoolByPathOrNone(json, DISABLE_BUILTIN_SECURITY_PATH).value_or(false);
         }
         EraseByPath(json, DISABLE_BUILTIN_SECURITY_PATH);
-        ctx.ExplicitEmptyDefaultGroups = CheckExplicitEmptyArrayByPathOrNone(json, DEFAULT_GROUPS_PATH).value_or(false);
-        ctx.ExplicitEmptyDefaultAccess = CheckExplicitEmptyArrayByPathOrNone(json, DEFAULT_ACCESS_PATH).value_or(false);
+        if (!ctx.ExplicitEmptyDefaultGroups) {
+            ctx.ExplicitEmptyDefaultGroups = CheckExplicitEmptyArrayByPathOrNone(json, DEFAULT_GROUPS_PATH).value_or(false);
+            EraseByPath(json, DEFAULT_GROUPS_PATH);
```

Comment:
Not sure that we should erase it, as far as we have it inside the real SecurityConfig